<launch>
<!-- start the bridge -->
   <param name="RIB_server_ip" value="192.168.178.42" type="str"/>
   <param name="RIB_port" value="18946" type="int"/>
   <param name="RIB_type" value="client" type="str"/>
   <node pkg="ros_igtl_bridge" type="ROS_IGTL_Bridge" name="ROS_IGTL_BRIDGE" output="screen">

   </node>
<!-- start the test -->
   <param name="test_pd" value="$(find ros_igtl_bridge)/testing/model.vtk" type="str"/>
   <param name="test_oct" value="$(find ros_igtl_bridge)/testing/oct_raw.cache" type="str"/>
   <node name="ROS_IGTL_TEST" pkg="ros_igtl_bridge" type="ROS_IGTL_Test"     output="screen">
   </node>
</launch>

